ui/admin/tests/e2e/README.md

```diff
@@ -0,0 +1,49 @@
+# Admin UI e2e tests
+
+This test suite tests the Boundary Admin UI in an end-to-end setting. It was designed to be run in a
+variety of environments as long as the appropriate environment variables are set. You can choose to
+spin up your own infrastructure or use [Enos](https://github.com/hashicorp/boundary/tree/main/enos)
+to generate it for you.
+
+The test suite uses the [Playwright](https://playwright.dev/) framework.
+
+## Getting Started
+Setup...
+```bash
+cd boundary-ui/ui/admin
+yarn install
+npx playwright install # this installs the browsers used by Playwright
```

Comment:
I will mark this as optional, since browsers will be install per demand anyways if they are not at running the test.

Reply:
Is it optional? I *think* I was greeted with a message to run this command before I could run tests.

Reply:
Documentation changed quite a bit since I put together e2e for Desktop, but at that time was 100% optional, and [following documentation right now](https://playwright.dev/docs/cli#install-browsers) seems to me it is.

But thinking twice... I does not hurt running it 😉